engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.capsule-content.controller:CapsuleContentController
+ *
+ * @requires $scope
+ * @requires $urlMatcherFactory
+ * @requires $location
+ * @requires $translate
+ * @requires $CapsuleContent
+ * @requires $AggregateTask
+ *
+ * @description
+ *   Provides the functionality for the capsule-content page.
+ */
+angular.module('Bastion.capsule-content').controller('CapsuleContentController',
+    ['$scope', '$urlMatcherFactory', '$location', 'translate', 'CapsuleContent', 'AggregateTask',
+    function ($scope, $urlMatcherFactory, $location, translate, CapsuleContent, AggregateTask) {
+
+        var urlMatcher = $urlMatcherFactory.compile("/smart_proxies/:capsuleId");
+        var capsuleId = urlMatcher.exec($location.path()).capsuleId;
+
+        var syncState = {
+            DEFAULT: 'DEFAULT',
+            SYNCING: 'SYNCING',
+            SYNC_TRIGGERED: 'SYNC_TRIGGERED',
+            CANCEL_TRIGGERED: 'CANCEL_TRIGGERED',
+            FAILURE: 'FAILURE',
+            set: function (state) {
+                return this.state = state;
+            },
+            is: function () {
+                return _.contains(arguments, this.state || this.DEFAULT);
+            }
+        }
+
+        function refreshSyncStatus() {
+            CapsuleContent.sync_status({id: capsuleId}, function (syncStatus) {
+                $scope.syncStatus = syncStatus;
+                if (syncStatus.active_sync_tasks.length > 0) {
```

Comment:
This will cause you linting errors due to requiring camelCase for JS. You can get around it via `syncStatus['active_sync_tasks']`